pages/api/account/account.js

```diff
@@ -0,0 +1,81 @@
+import logger from "@config/logger";
+import { authOptions } from "../auth/[...nextauth]";
+import { getServerSession } from "next-auth/next";
+import connectMongo from "@config/mongo";
+import { Account } from "@models/index";
+
+export default async function handler(req, res) {
+  const session = await getServerSession(req, res, authOptions);
+
+  if (!session) {
+    res.status(401).json({ message: "You must be logged in." });
+    return;
+  }
+
+  if (req.method !== "GET") {
+    return res.status(400).json({ error: "Invalid request: GET request required" });
+  }
+
+  try {
+    const data = await getAccountByProviderAccountId(session.user.id);
+    res.status(200).json(data);
+  } catch (error) {
+    logger.error('Error in handler:', error);
```

Comment:
The exception is the first parameter...

```suggestion
    logger.error(error, 'Error in Account handler');
```